client/src/components/Monument/Gallery/Gallery.js

```diff
@@ -36,6 +36,12 @@ export default class Gallery extends React.Component {
     }
 
     animate(index) {
+        const { images } = this.props;
+        if (images.length <= 1) {
+            this.setState({animating: false});
+            return;
+        }
+
```

Comment:
The image animation was occurring even though there was only one image, which looked funky.